File-Formats/installation-and-upgrade/System-Requirements.md

```diff
@@ -13,7 +13,7 @@ The system requirements for using our Syncfusion FileFormats platform are as fol
 
 ### Operating Systems
 
-Syncfusion File Formats platform can be installed on any of the following operating systems:
+Syncfusion File Format libraies are supported in any of the following operating systems:
```

Comment:
Syncfusion File Format libraries are supported on any of the following operating systems:

Reply:
Addressed